tox.ini

```diff
@@ -25,7 +25,7 @@ deps =
 deps = pre-commit
 commands = pre-commit run --all-files --show-diff-on-failure {posargs}
 
-[testenv:py38-pytestmaster]
+[testenv:py{38,310}-pytestmaster]
```

Comment:
that would not cover `py310-pytestlatest`, which is the only env with python 3.10. 

```suggestion
[testenv:py38-pytestmaster]
```

Let's add a new separate section:

```ini
[testenv:py310-pytestlatest]
# allow failures of tests run with unstable python 3.10
ignore_outcome = true
````